backend/Dockerfiles/Dockerfile.veracode

```diff
@@ -76,28 +86,32 @@ RUN mkdir -p /node && \
     mv /node/node-v$NODEVER-linux-x64 /node/node && \
     rm /node/node.tar.gz
 
-FROM python:3.9-bullseye as ruby-builder
+FROM php@${PHP_IMG_VER} as php-builder
 
-ARG RUBYVER=2.7.2
-ARG RUBYSHA=3f50b100fb52cdf315fa17f41ae2e2538bb0c45abd9a6c569fd70ac851d61b2a
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-RUN mkdir -p /ruby && \
-    echo "$RUBYSHA /ruby/ruby.tar.bz" > /ruby_checksum.txt && \
-    curl https://rvm_io.global.ssl.fastly.net/binaries/debian/10/x86_64/ruby-$RUBYVER.tar.bz2 -L -o /ruby/ruby.tar.bz && \
-    sha256sum -c /ruby_checksum.txt && \
-    tar -xjvf /ruby/ruby.tar.bz -C /ruby && \
-    mv /ruby/ruby-$RUBYVER /ruby/ruby && \
-    rm /ruby/ruby.tar.bz
```

Comment:
Had to install ruby with apt-get instead, because the official website does not have binaries for Debian bookworm yet.